Formula/crystal-lang.rb

```diff
@@ -42,7 +42,7 @@ def install
       ENV["CRYSTAL_CONFIG_VERSION"] = version
     end
 
-    ENV["CRYSTAL_CONFIG_PATH"] = prefix/"src:libs:lib"
+    ENV["CRYSTAL_CONFIG_PATH"] = prefix/"src:lib:libs"
```

Comment:
Because since shards 0.7.0, which Crystal now includes with this formula (well, it includes 0.7.1 now) shards installs dependencies in the `lib` directory:

https://github.com/crystal-lang/shards/releases/tag/v0.7.0

> Dependencies are installed in the lib directory

So `lib` should be preferred over `libs`. I still left `libs`, I'm doing this for just one release to ease upgrade.